cypress/support/ComboiosDePortugal.js

```diff
@@ -0,0 +1,73 @@
+
+const pageHeader = '[class="navbar-brand"]';
+const fromInputFiled = '[title="From "]';
+const toInputFiled = '#arrival-date';
+const departDate = '[name="departDate"]';
+const fromdropdown = '[class="typeahead dropdown-menu"] a';
+const dropdown = '[class="typeahead dropdown-menu"] a';
+const currentDay  = '[class*="day--today"]';
+const returnDate= '[name="returnDate"]';
+const submitButton ='[value="Submit »"]';
+
+const getFullDate = {
```

Comment:
Hi @HACKERANIL 
Please store these methods in the utilities file and read it from the spec file.